docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -70,18 +63,18 @@
 |==============================================
 |Name |Type |Description |Required
 
-|`comments` |xref:exception-item-comment-schema[comment][] a|Array of xref:exception-item-comment-schema[`comment`] objects. |No, defaults to empty array.
+|`comments` |xref:exception-{endpoint-artifact-api-doc-name}-item-comment-schema[comment][] a|Array of xref:exception-{endpoint-artifact-api-doc-name}-item-comment-schema[`comment`] objects. |No, defaults to empty array.
 |`description` |String |Describes the {endpoint-artifact-name-single} item. |Yes
-|`entries` |xref:exception-item-entry-schema[entry][] |Array containing the {endpoint-artifact-name-single} query conditions. See xref:exception-item-entry-schema[] for more details. |Yes.
+|`entries` |xref:exception-{endpoint-artifact-api-doc-name}-item-entry-schema[entry][] |Array containing the {endpoint-artifact-name-single} query conditions. See xref:exception-{endpoint-artifact-api-doc-name}-item-entry-schema[] for more details. |Yes.
 |`list_id` |String a|Must be set to `pass:a[{endpoint-artifact-list-id}]`. |Yes
 |`item_id` |String |Unique identifier of the exception item. |No, automatically
 created when it is not provided.
 |`meta` |Object |Placeholder for metadata about the exception item. |No
 |`name` |String |The {endpoint-artifact-name-single} name. |Yes
 |`namespace_type` |String a|Must be set to `agnostic` for endpoint artifacts. |Yes
-|`os_types` |xref:exception-item-os-types[os_type][] |Array of xref:exception-item-os-types[]. | Yes
+|`os_types` |xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[os_type][] |Array of xref:exception-{endpoint-artifact-api-doc-name}-item-os-types[]. | Yes
```

Comment:
I've updated `...item-os-types[]` to `item-os-types[os_types] values.` in my PR.